### PR TITLE
Add missing entries on allowlist

### PR DESCRIPTION
**Pull request by slord399**
All according to API
https://api.vrchat.cloud/api/1/config

### CODE REVIEW

**Review by slord399 — COMMENTED** [review]
Add more missing entry as per as 16th October 2023.

**Comment by Miner28** [comment]
Some of those were intentionally left out from what i remember.

**Comment by slord399** [comment]
I agree but people seems want to see it as full list as per as related canny.

File: Docs/docs/worlds/udon/image-loading.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ And only certain domains domains are allowed. If a domain is not on the list, im
 - Reddit (`i.redd.it`)
 - Twitter (`pbs.twimg.com`)
 - VRChat (`assets.vrchat.com`)
+- Virtual Market (`*.v-market.work`,`*.vket.chat`)
+- Gugenka (`*.xr-marketplace.com`)
+- Poly (`*.poly.jp`)
+- VazaR (`*.vazar.jp`)
+- CatsUDON (`*.catsudon.net`)
```
Comment (Faxmashine) [inline Docs/docs/worlds/udon/image-loading.md:51]:
```suggestion
- Poly (`*.poly.jp`)
```

File: Docs/docs/worlds/udon/string-loading.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ The following URLs are available:
 * GitHub (`*.github.io`)
 * Pastebin (`pastebin.com`)
 * Github Gist (`gist.githubusercontent.com`)
+* Gugenka  (`*.xr-marketplace.com`)
+* Virtual Market (`*.v-market.work`,`*.vket.chat`)
+* Fynn (`api.fynn.ai`,`api-dev.fynn.ai`)
+* Poly  (`*.poly.jp`)
+* VazaR  (`*.vazar.jp`)
+* CatsUDON  (`*.catsudon.net`)
```
Comment (Faxmashine) [inline Docs/docs/worlds/udon/string-loading.md:30]:
```suggestion
* Poly  (`*.poly.jp`)
```

File: Docs/docs/worlds/udon/video-players/www-whitelist.md [hunk 3]
```diff
@@ -48,4 +48,15 @@ The services listed below are inherently trusted and are permitted with our defa
...
+| Virtual Market | `music4.cdn.vket.com`,`*.v-market.work`,`*.vket.chat`,`movie.vketcdn.com` |
+| Gugenka | `*.xr-marketplace.com` |
+| Twist Events | `twisteventsasia.com` |
+| Unity Technologies | `stream.broadcast.unity.com` |
+| Furality Sylva | `*.furality.online` |
+| FoxCDN | `live.foxcdn.io` |
+| Poly | `*.poly.jp` |
+| VazaR | `*.vazar.jp` |
+| CatsUDON | `*.catsudon.net` |
+| Virtual Okinawa | `virtualokinawa-on-demand.contents.work` |
```
Comment (Faxmashine) [inline Docs/docs/worlds/udon/video-players/www-whitelist.md:62]:
```suggestion
| Mixcloud | `mixcloud.com` |
| Poly | `*.poly.jp` |
```

**Review by Faxmashine — CHANGES_REQUESTED** [review]
Hi @slord399 ! Thanks again for the PR. Would you mind accepting the following changes?

The purpose of this list is to let world creators know what streaming services can be used without enabling 'trusted URLs'. Sites like Poly and Mixcloud should be in that list. Thank you for adding them!

The other sites you listed aren't streaming services, and world creators shouldn't rely on being able to access them.

**Comment by Miner28** [comment]
@Faxmashine Or another option I think would be to separate this list into 2 lists.
1. List of streaming services as it is right now that are trusted
2. List of all services that are whitelisted for privacy-concerned users to see the full list.

If that gets added, it could be hidden under a drop-down section.
However I don't know if that's really required/point of this page.

**Comment by slord399** [comment]
Yeah, I don't mind remove non-generic use service, but it can be changed in feature anyway.
+
Privacy concern and missing some site be not good on local regularation matter.
So I would bit to make other page which state all the untrusted URLs available to cope with local regularations which state you requred to have consent before actually using it. 
(As entry missing from docs means user may not consent to use these service anyway. Let's make it GDPR friendly at least.)

**Review by slord399 — COMMENTED** [review]
at leasat revised.
(anyway, still request to have separate docs for list all domains which used when untrusted URLs is opt-out.)

**Review by Faxmashine — APPROVED** [review]
Thank you for the changes!

**Pull request merged**